cves/kernel/CVE-2018-20854.yml

```diff
@@ -320,8 +321,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: false
```

Comment:
this should be true based on this 
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=6acb47d1a318e5b3b7115354ebc4ea060c59d3a1 which was found here https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-20854 I think. The following people Signed-off:
* Gustavo A. R. Silva <gustavo@embeddedor.com>
*  David S. Miller <davem@davemloft.net>

Reply:
Thanks, crazy I didn't notice that.